### DISCUSSION TITLE
Hyprland randomly delete /tmp/hypr folder

### DISCUSSION THREAD

**Discussion by filippoaceto22** [hyprwm/Hyprland#4079] [General]
I have two machine with archlinux and randomly delete /tmp/hypr and recreate It empty, only with logs.
Hyprland so crash because does not found sockets.
I don t understant why.
In each machine the only things i can say Is that i use Eclipse ide for Java developments.

Thanks

**Comment by filippoaceto22** [comment]
up

**Comment by ildar** [comment]
that could be the `tmp cleanup service`

**Reply by filippoaceto22** [reply]
I need to investigate, but I don't see that directory being touched
